src/test/java/com/palpal/dealightbe/domain/item/domain/ItemRepositoryTest.java

```diff
@@ -0,0 +1,420 @@
+package com.palpal.dealightbe.domain.item.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Slice;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.palpal.dealightbe.domain.address.domain.Address;
+import com.palpal.dealightbe.domain.address.domain.AddressRepository;
+import com.palpal.dealightbe.domain.member.domain.Member;
+import com.palpal.dealightbe.domain.member.domain.MemberRepository;
+import com.palpal.dealightbe.domain.store.domain.DayOff;
+import com.palpal.dealightbe.domain.store.domain.Store;
+import com.palpal.dealightbe.domain.store.domain.StoreRepository;
+import com.palpal.dealightbe.domain.store.domain.StoreStatus;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+class ItemRepositoryTest {
+
+	@Autowired
+	private MemberRepository memberRepository;
+
+	@Autowired
+	private AddressRepository addressRepository;
+
+	@Autowired
+	private StoreRepository storeRepository;
+
+	@Autowired
+	private ItemRepository itemRepository;
+
+	private Item item1;
+	private Item item2;
+	private Item item3;
+	private Item item4;
+	private Item item5;
+	private Item item6;
+
+	@BeforeEach
+	void setUp() {
+		String storeDefaultImageUrl = "https://fake-image.com/store.png";
+
+		Member member1 = Member.builder()
+			.nickName("닉네임")
+			.realName("이름")
+			.phoneNumber("010010101")
+			.provider("kakao")
+			.providerId(123L)
+			.build();
```

Comment:
Member도 그렇고 Item도 그렇고 이런식으로 객체 생성하는 메소드 빼도 될 것 같아요!


```suggestion
		Member member1 = createMember(providerId);
```